kuma/core/context_processors.py

```diff
@@ -62,6 +62,19 @@ def next_url(request):
     def inner():
         if hasattr(request, "path"):
             if request.GET.get("next"):
+                # Make a special exception for 'next' URLs that are absolute but
+                # who's host is in a known list.
+                if (
+                    "://" in request.GET["next"]
+                    and settings.DEBUG
+                    and settings.ADDITIONAL_NEXT_URL_ALLOWED_HOSTS
+                    and settings.ADDITIONAL_NEXT_URL_ALLOWED_HOSTS
+                    in request.GET["next"]
```

Comment:
tiny nit: wouldn't the `in` check allow the host to appear _somewhere_ in the URL, thereby potentially letting any host through (only in DEBUG, hence a nit)?

Reply:
I thought about that but then I realized that it's overkill since it's only ever happening with `settings.DEBUG===True`